src/cmd/root.go

```diff
@@ -162,7 +162,7 @@ func preRun(cmd *cobra.Command, args []string) error {
 				return err
 			}
 
-			return errors.New("TOOLBOX_PATH not set")
+			return errors.New("this is not a Toolbox container \n Toolbox is only meant to work with Toolbox containers.")
```

Comment:
Instead of duplicating the error message, and running the risk of copy-paste mistakes, we should have a single function (in `src/cmd/utils.go`) that returns the error.